src/DataFetcher.py

```diff
@@ -8,322 +9,328 @@
 from src.YahooFinance import YahooFinanceQuoteSummary, YahooFinanceQuoteSummaryModule
 from threading import Lock
 
+logger = logging.getLogger("IsThisStockGood")
+
+
 def fetchDataForTickerSymbol(ticker):
```

Comment:
Lines 15-131 seem to all just be whitespace changes. Let's revert those if we can.